acceptance-tests/helpers/brokers/env.go

```diff
@@ -31,7 +31,7 @@ func (b Broker) env() []apps.EnvVar {
 		switch {
 		case ok:
 			result = append(result, apps.EnvVar{Name: name, Value: val})
-		case !ok && required:
+		case required:
```

Comment:
This will cause it to always complain for all required env vars even if they are set

Reply:
The condition `!ok` in the second case statement is redundant because the switch statement already implicitly checks for the `!ok` condition when the first case `ok` is not met.
The second case `!ok && required` explicitly checks `!ok`, but since the control only reaches this case when `ok` is false, the `!ok` part is redundant. The `required` condition alone is sufficient.

Reply:
I see -that makes sense, but it does make it harder to reason (as in you have to think about the other case)